isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -1509,6 +1634,64 @@ namespace Isis {
   }
 
 
+  /**
+   * Outputs lidar data to a csv file.
+   *
+   * @return bool If the point data was successfully output.
+   */
+  bool BundleSolutionInfo::outputLidarCSV() {
```

Comment:
Where is it decided if this does / does not get output?

Reply:
Actually would not mind some clarification on this!

Reply:
This gets called in jigsaw. See:

https://github.com/USGS-Astrogeology/ISIS3/blob/LroLidar_Infrastructure/isis/src/control/apps/jigsaw/main.cpp#L126

https://github.com/USGS-Astrogeology/ISIS3/blob/LroLidar_Infrastructure/isis/src/control/apps/jigsaw/jigsaw.xml#L1436

So this will get called by the code for #4705

Reply:
I was trying to look WITHIN the PR for where it was called.. wow.. that's on me!